colab/static/js/subscriptions.js

```diff
@@ -0,0 +1,32 @@
+$(function (){
+
+  ENTER_BUTTON = 13;
+  searchMaillingLists();
+
+  $('#search_list').on('click', function(event) {
+    searchMaillingLists();
+  });
+
+  $('#list_name').on('keypress', function(event) {
```

Comment:
Yeah, this feature had this little 'bug' because keypress event.. I never saw this input event before, very nice! I'll also use it in another project I maintain haha!
Thanks, @chaws!